pkgs/build-support/cc-wrapper/utils.sh

```diff
@@ -25,7 +25,10 @@ mangleVarBool() {
     for infix in "${role_infixes[@]}"; do
         local inputVar="${var/+/${infix}}"
         if [ -v "$inputVar" ]; then
-            let "${outputVar} |= ${!inputVar}"
```

Comment:
I think one can directly do `"${!inputVar:-0}"`?

Reply:
Ah, didn't know that!